### PR TITLE
Feature/codelist options

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

File: config/resources/domain.json [hunk 1]
```diff
@@ -79,8 +79,9 @@
       "relationships": {
         "concepts": {
           "predicate": "skos:inScheme",
-          "target": "concept",
-          "cardinality": "many"
+          "target": "skosConcept",
```
Comment (nvdk) [inline config/resources/domain.json:82]: if this wasn't here before, we will probably need a migration to migrate the old data?

Comment (Asergey91) [reply config/resources/domain.json:82]: I mean we didnt have any data that used the previous relationship

Comment (Asergey91) [reply config/resources/domain.json:82]: I guess Ideally we need codelist options inherit from skosConcept and then we would need to run a migration

Comment (Asergey91) [reply config/resources/domain.json:82]: but actually im not sure how inheritance and relationships work together.... gonna look at some triples I guess

Comment (lagartoverde) [reply config/resources/domain.json:82]: I guess if the codelist options are no different than a concept so they shouldn't change, just have to merge then in concepts and expose a single endpoint

Comment (Asergey91) [reply config/resources/domain.json:82]: options

Comment (Asergey91) [reply config/resources/domain.json:82]: no migrations needed I dont think

**Pull request merged**